internal/services/wallets/wallets_testnet.go

```diff
@@ -30,4 +30,11 @@ var TestnetWallets = []data.Wallet{
 		Assets:      assets.AllAssetsTestnet,
 		UserManaged: true,
 	},
+	{
+		Name:           "SDP Embedded Wallet",
+		DeepLinkSchema: "https://stellar.org",
+		Homepage:       "https://stellar.org",
+		Assets:         assets.AllAssetsTestnet,
+		Embedded:       true,
+	},
```

Comment:
[Not for this PR] WDYT about shielding the embedded wallet behind a feature flag? This may help us merge the C-accounts branch earlier.